mappings/net/minecraft/entity/decoration/painting/PaintingEntity.mapping

```diff
@@ -1 +1,25 @@
 CLASS net/minecraft/unmapped/C_xypmuulf net/minecraft/entity/decoration/painting/PaintingEntity
+	FIELD f_cnshsoix variantKey Lnet/minecraft/unmapped/C_xhhleach;
```

Comment:
Reviewed before seeing the build result, oops

```suggestion
	FIELD f_cnshsoix VARIANT_KEY Lnet/minecraft/unmapped/C_xhhleach;
```
As apparently it's a static final field.